tests/test_timers.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import json
+from application import create_app, db
+from application.models.timer import Timer
+
+
+class TestTimers(unittest.TestCase):
+    def setUp(self):
+        self.app = create_app('testing')
+        self.test_app = self.app.test_client()
+        with self.app.app_context():
+            db.create_all()
+
+    def tearDown(self):
+        with self.app.app_context():
+            # drop all tables
+            db.session.remove()
+            db.drop_all()
+
+    def test_get_all_timers(self):
+        timer1 = Timer(work_interval='25:00', rest_interval='5:00')
+        timer2 = Timer(work_interval='45:00', rest_interval='15:00')
+        with self.app.app_context():
+            db.session.add(timer1)
+            db.session.add(timer2)
+            db.session.commit()
+
+        # create a user
```

Comment:
These comments are super helpful. They also show good dev empathy IMHO.

Reply:
Yas it's good for us FE's too!